services/pull_request/parse_repository.rb

```diff
@@ -0,0 +1,17 @@
+module PullRequest
+  class ParseRepository
+    attr_accessor :repo_name, :issue_number
+
+    def initialize(repo)
+      @repo = repo
+    end
+
+    def call
+      @repo.slice!('https://github.com/')
```

Comment:
@mbroderson it would be better to move url to some static or another place (config, etc.)